app/views/scheduled_messages/index.html.erb

```diff
@@ -46,6 +46,12 @@
             <span>Test</span>
           <% end %>
         </td>
+        <td>
+          <a class="button button-delete" data-confirm="Are you sure you want to delete this message?" rel="nofollow" data-method="delete" href="/scheduled_messages/<%= message.id %>">
```

Comment:
nice call on the js confirm!

let's use a rail `link_to` helper rather than an `a` tag:

http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

and let's use the named rails route instead of creating the URL manually: 

http://api.rubyonrails.org/classes/ActionDispatch/Routing.html#module-ActionDispatch::Routing-label-Named+routes